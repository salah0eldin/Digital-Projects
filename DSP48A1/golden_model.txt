2936 15532 2093052932 7674 15845 104 1910510412 1 -> 15532 45601952 2007068441 2007068441
29903 20602 1395421043 3769 13822 45 41951565 0 -> 20602 616061606 2011482681 2011482681
28562 9927 1893353827 13485 16651 136 488440265 1 -> 9927 283534974 2011482681 2011482681
15258 9300 96818719 7352 22843 0 1508261892 1 -> 9300 141899400 0 0
6941 17238 1160453353 32168 18938 204 861896214 1 -> 17238 119648958 1160453353 1160453353
13894 1852 474971215 379 4105 77 1084372637 0 -> 1852 25731688 500702903 500702903
21432 30097 1606385346 29464 21462 195 80417552 0 -> 30097 645038904 1791261928 1791261928
12850 20390 1191540766 15857 7186 74 432284602 0 -> 20390 262011500 -712443440 -712443440
31147 18862 430924534 10533 9874 96 202840293 1 -> 18862 587494714 32 32
7839 20316 1578118371 15892 20477 133 1315793094 1 -> 20316 159257124 1156535970 1156535970
