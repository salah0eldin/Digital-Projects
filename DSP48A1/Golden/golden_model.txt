23e0 4aa9 9ef5152 ce1 5054 76 5d484377 0 -> ffffc238 fffffffff7579900 0 8d4eaa867
3cb2 45f8 2ad268ee 694b 427f fffffff4 12fa0c66 1 -> 2353 86003b6 0 12fa0c67
4c6e 4771 7ecd3b54 7370 2236 ffffffd8 48c953ba 1 -> 2bff d229b92 0 12fa0c67
19c5 994 32147360 7835 3f50 31 262c461a 0 -> 81c9 d1080ad 0 d1080ac
25c7 6e44 297030f6 aa4 7163 ffffff93 3b6d1590 0 -> 78e8 11d78458 0 15a3855c
5ec5 14b8 cd201d6 3694 107d 4 1f6390e 0 -> 14b8 7ab8198 0 1f6390e
64ba 3ec1 6390395d 396d 2bcb 46 6f00117 0 -> 3ec1 18b0fc3a 0 8e63a25
3469 25d6 272718cb 7f64 3d03 ffffff88 236e77ca 0 -> 25d6 7befcc6 0 8e63a25
20a9 29d0 44797ff6 47e5 fd8 ffffffbf 3ca43ca8 0 -> 71b5 e81b07d 0 8d6ee812
d70 1e3a 38e43e9d 420f 76c1 39 600700c9 1 -> 6049 50dd4f0 0 927cbd01
